### DISCUSSION TITLE
Welcome to bet365-re-js Discussions!

### DISCUSSION THREAD

**Discussion by re-bet365** [re-bet365/bet365-re-js#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by carlreiser2** [comment]
Hey do you have any idea how the X-Net-Sync-Term gets generated in the obfuscated code and gets attached to bet365 network requests? If we knew this wouldn't we be able to make the network requests without the need of puppeteer/playwright?

**Reply by re-bet365** [reply]
That is one of the things in our TODO list.
Please bear with us until everything is figured out one at a time :)
